Source/ZoomNet/Models/DateObject.cs

```diff
@@ -0,0 +1,30 @@
+using System.Text.Json.Serialization;
+
+namespace ZoomNet.Models
+{
+	/// <summary>
+	/// Date Object.
+	/// </summary>
+	public class DateObject
```

Comment:
This class contains data about daily usage right? It should probably be called something like `DailyUsageSummary`.